core/sdk/src/framework/ripple_contract.rs

```diff
@@ -113,6 +113,8 @@ pub enum RippleContract {
     Metrics,
     /// Contract for Extensions to recieve Telemetry events from Main
     OperationalMetricListener,
+    /// Contract for Extensions to stand in for a WebSocket server based service provider
+    MockWebsocketServer,
```

Comment:
Whats your thought on making `mock` as a common contract with MockTestAdjective?

you can then do
`ws.mock`
`http.mock`
`grpc.mock`
`com_rpc.mock` -

Reply:
Current impl would work for mocking a single websocket but we can't probably combine this with a PubSub Extension mock(which is on the horizon). Main has to have a way to detect the right extension(and processor) to send the request to.

Reply:
Sounds like a good idea. But maybe `mock_server` would be the noun?

Reply:
Done.